orienteer-core/src/main/java/org/orienteer/core/web/BasePage.java

```diff
@@ -152,8 +154,13 @@ public void renderHead(IHeaderResponse response) {
 //		response.render(new PriorityHeaderItem(JavaScriptHeaderItem.forReference(METISMENU_JS)));
 		// enabling metisMenu for secondary level menus
 //		response.render(OnDomReadyHeaderItem.forScript("$(\".metismenu\").metisMenu({toggle: true});"));
+		response.render(OnDomReadyHeaderItem.forScript("document.body.setAttribute('class', 'app "+getBodyAppSubClasses()+"');"));
```

Comment:
Why it's needed in this way? Why styles can't be statically adjusted on corresponding HTML?

Reply:
It is simplest method to change body style in child classes. If we use static html - we should make body component as wicket component. This is change hierarchy and unpredictably change other components behaviour.